game/scripts/npc/abilities/bristleback_quill_spray.txt

```diff
@@ -58,7 +58,7 @@
       "05"
       {
         "var_type"                                        "FIELD_FLOAT"
-        "max_damage"                                      "550.0 550.0 550.0 550.0 1100.0 2200.0"
+        "max_damage"                                      "150.0 275.0 400.0 550.0 825.0 1100.0"
```

Comment:
This is all because of bottle, you know. We said "let's keep Dota 2 vanilla the same and just build on top of it". Then Infinity Bottles happened. Dota 2 vanilla stats are going to have to change or we are going to have to reduce the impact of bottles.